### ISSUE TITLE
configure script requires Python argparse module (not available in Python 2.6)

### ISSUE THREAD

**Issue by boegel** [Fahey-McLay/xalt#14]
I see `configure` failing with:

```
xalt_configuration_report.C:2:24: fatal error: xalt_regex.h: No such file or directory
 #include "xalt_regex.h"
                        ^
compilation terminated.
make[1]: *** [xalt_configuration_report.d] Error 1
make[1]: Leaving directory `/home/kehoste/build/XALT/1.5.3-devel/GCC-4.9.3-2.25/xalt-1.5.3-devel/compiled'
make: *** [build_compiled] Error 2
```

the `xalt_regex.h` is not being generated because of this:

```
Traceback (most recent call last):
  File "./src/build_xalt_regex.py", line 26, in <module>
    import os, sys, re, argparse
ImportError: No module named argparse
Traceback (most recent call last):
  File "./src/build_syshost_routine.py", line 28, in <module>
    import os, sys, re, argparse, json
ImportError: No module named argparse
```

The `argparse` module is not available (by default) for Python 2.6 (cfr. https://docs.python.org/2/library/argparse.html), so `configure` should check for it (or not use `argparse`).

**Comment by rtmclay** [comment]
Please test this commit.  This new configure script puts argparse.py in PYTHONPATH if python 2.6 is being used.

**Comment by boegel** [comment]
This is indeed fixed with 5187e5b85859b48a81c7a0ecc403e5dedd6abc87, XALT 1.5.5-devel which has this included installs fine on a system that has Python 2.6 without `argparse`, thanks @rtmclay!
